cfgen/cfgen

```diff
@@ -1470,11 +1454,51 @@ def pool_drives(m0conf: Dict[Oid, Any],
             'Pool {!r}: some of disk_refs refer to the same disk'.format(
                 pool_desc['name'])
 
+    return drives
+
+
+def pool_drives(m0conf: Dict[Oid, Any],
+                pool_desc: Dict[str, Any]) -> List[Oid]:
+    ptype = pool_type(pool_desc)
+
+    if ptype is PoolT.dix:
+        cas = [(svc_id, m0conf[svc_id].ctrl_id)
+               for svc_id in cluster_svcs(m0conf, SvcT.M0_CST_CAS)]
+
+        return [ConfDrive.build(m0conf, ctrl_id,
+                                ConfSdev.build(m0conf, svc_id,
+                                               Disk(path='/dev/null',
+                                                    size=1024,
+                                                    blksize=1)))
+                for (svc_id, ctrl_id) in cas]
+
+    if ptype is PoolT.md:
+        return [m0conf[ctrl_id].drives[0]
+                for ctrl_id in m0conf if ctrl_id.type is ObjT.controller]
+
+    assert ptype is PoolT.sns
+    drives = pool_drives_from_references(m0conf, pool_desc)
+    if not drives:
+        drives = pool_drives_with_nodes(m0conf)
+
+    return [drive_id for drive_id, _ in drives]
+
+
+def check_drive_multiple_references(m0conf: Dict[Oid, Any],
```

Comment:
1. Please specify the return type
2. Please add the docstring to this function

Reply:
Addressed.